### PR TITLE
doc: added AES crypto sample

### PR DESCRIPTION
**Pull request by peknis**
Converted the sample in the Infocenter to rst.

Signed-off-by: Pekka Niskanen <pekka.niskanen@nordicsemi.no>

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -76,6 +76,7 @@ Kconfig*                                  @tejlmand
 /samples/bluetooth/                       @joerchan @lemrey @carlescufi
 /samples/bluetooth/mesh/                  @trond-snekvik @joerchan
 /samples/bootloader/                      @hakonfam @ioannisg
+/samples/crypto/aes/					  @peknis
```
Comment (b-gent) [inline CODEOWNERS:79]: FYI, this is done only for CI issues, obviously a different code owner should be put here

File: doc/nrf/samples.rst [hunk 2]
```diff
@@ -18,6 +18,15 @@ In addition, the |NCS| provides the following samples that showcase the use of a
    ../../samples/bluetooth/*/README
    ../../samples/bluetooth/mesh/*/README
 
+.. _crypto_aes_samples:
```
Comment (b-gent) [inline doc/nrf/samples.rst:21]:
```suggestion
.. _crypto_samples:
```

File: samples/crypto/aes/README.rst [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+.. include:: /includes/build_and_run.txt
+
+.. _crypto_aes_testing:
+
+Testing different modes
+=======================
+
+The following process is applicable for testing AES CTR, AES CBC_MAC and AES CCM modes:
+
+1. Skip this step if you are using an RTT Viewer. By default, the crypto examples are configured to use RTT for logging. Start a terminal emulator like PuTTY and connect to the used COM port with the following UART settings:
```
Comment (b-gent) [inline samples/crypto/aes/README.rst:54]: New sentence from new line :) (everywhere)

**Review by b-gent — CHANGES_REQUESTED** [review]

File: samples/crypto/aes/README.rst [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+* AES API for CTR
+* AES API for CBC-MAC
+* AEAD API for CCM
+* AES API and AEAD API for all supported AES modes.
+
+.. note::
+   These examples can run in software or hardware, depending on the supported features of your SoC.
+   They use the default backend for the specific platform, but you can test them using different backends.
+
+The source code and the project file of the sample are in ``<InstallFolder>\examples\crypto\nrf_crypto\aes``.
```
Comment (b-gent) [inline samples/crypto/aes/README.rst:28]: This is duplicated information from line 43

**Review by b-gent — CHANGES_REQUESTED** [review]

**Comment by b-gent** [comment]
A Dependencies section would be nice at the end as per the sample template:
https://github.com/nrfconnect/sdk-nrf/blob/master/doc/nrf/templates/sample_README.rst#dependencies

It is not mandatory but most samples have it, if you can list dependencies, please do it.

**Comment by peknis** [comment]
> A Dependencies section would be nice at the end as per the sample template:
> https://github.com/nrfconnect/sdk-nrf/blob/master/doc/nrf/templates/sample_README.rst#dependencies
> 
> It is not mandatory but most samples have it, if you can list dependencies, please do it.

I could not tell what sort of dependencies this sample has. How would I know which libraries this uses?

File: samples/crypto/aes/README.rst [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/crypto/aes`
+
+.. include:: /includes/build_and_run.txt
+
+.. _crypto_aes_testing:
+
+Testing different modes
```
Comment (frkv) [inline samples/crypto/aes/README.rst:47]: Needs replacement pointing to logging in nRF Connect SDK

File: samples/crypto/aes/README.rst [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+
+Testing all AES modes
+=====================
+
+To test All AES modes using command-line interface, perform the following steps:
+
+1. Compile and program the application.
+#. Start a terminal emulator like PuTTY (recommended) with the following Terminal settings, and establish the connection:
+
+   * Serial: for UART or USB CDC ACM transport.
```
Comment (frkv) [inline samples/crypto/aes/README.rst:73]: This needs to be converted into a sanity check mechanism of sorts

**Comment by peknis** [comment]
This was created just to be used as a template when porting crypto docs from Infocenter to NCS. Not needed anymore.

**Pull request closed without merging**